#!/bin/sh
# debian/patches/no_scrollkeeper-update.dpatch
# 

#  All lines beginning with `## DP:' are a description of the patch.
## DP: avoid `scrollkeeper-update` during "make install"

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects an argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad src/conglomerate-0.7.11/omf.make conglomerate-0.7.11/omf.make
--- src/conglomerate-0.7.11/omf.make	2004-01-18
+++ conglomerate-0.7.11/omf.make	2004-01-18
@@ -42,7 +42,7 @@
 	for file in $(omfincfile); do \
 		$(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(docdir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
+	##-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
